lib/linters/yamllint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -ou pipefail
+
+declare COMMON_LIB_PATH=""
+COMMON_LIB_PATH="$(dirname "$(dirname "${BASH_SOURCE[0]}")")/common.sh"
+
+#shellcheck disable=SC1090
+source "${COMMON_LIB_PATH}"
+
+WORKING_DIR="${2}"
+
+# TODO - https://yamllint.readthedocs.io/en/stable/configuration.html
+
+handle()
+{
+  local RC=0
+  local FILENAME="${1}"
+  local USERS_YAML_CONFIG_PATH="${WORKING_DIR}/.yamllintrc"
```

Comment:
yamllint's exclusion file is .yamllint not .yamllintrc
https://yamllint.readthedocs.io/en/stable/configuration.html